package.json

```diff
@@ -7,11 +7,12 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
```

Comment:
Fix

```json
{
  "main": "./dist/react-align.umd.js",
  "module": "./dist/react-align.es.js",
  "types": "./dist/index.d.ts",
  "exports": {
    ".": {
      "import": "./dist/index.es.js",
      "require": "./dist/index.umd.js"
    }
  } 
}
```

Refer to https://github.com/reearth/quickjs-emscripten-sync/blob/main/package.json

Reply:
I have already changed these around a little bit in PR 3. But the build output I am getting I have tried to mirror the old output as much as possible.
I am producing commonJs + its map and ES modules + its map. Should I build UMD instead of commonJS?
Also, the build output doesn't have index.XX.js files. Only react-align.XX.js. I couldn't figure out how to output the index, but the example project still works so I assumed they maybe aren't necessary.

Reply:
Is UMD preferred over commonJS? In PR3 I am updating here and the build, and trying to output the same as before moving to Vite, which was outputing commonJS and ES modules. The only thing I can't do that the old build was doing was the index.XX.js files, but the example project still runs fine so maybe they aren't needed?

Reply:
UMD arelady supports CommonJS, so we do not need to build JS for CommonJS. quickjs-emscripten-sync already deletes CommonJS builds and use UMD build instead.

In the future, CommonJS will be one of the module systems of the past that existed a long time ago on a day of the week when AMD is no longer in use, so it makes sense to group them into UMD.

In fact, Vite's documentation already unifies it with UMD.